src/main/java/lotto/manageLotto/CompareLottos.java

```diff
@@ -0,0 +1,54 @@
+package lotto.manageLotto;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CompareLottos {
+
+  public static Map<String, Integer> compareNums(List<List<Integer>> myLottos, List<Integer> winLotto, int bonusNum) {
```

Comment:
```suggestion
  public static Map<String, Integer> compareNums(List<Lotto> myLottos, List<Integer> winLotto, int bonusNum) {
```

Lotto객체를 활용하지 못하는 것 같습니다.. Lotto 객체를 활용해주세요